lib/temperature_and_humidity_sensor_accessory.js

```diff
@@ -0,0 +1,92 @@
+const BaseAccessory = require('./base_accessory')
+
+let Accessory;
+let Service;
+let Characteristic;
+
+class TemperatureAndHumiditySensorAccessory extends BaseAccessory {
+
+    constructor(platform, homebridgeAccessory, deviceConfig) {
+        ({ Accessory, Characteristic, Service } = platform.api.hap);
+        super(
+            platform,
+            homebridgeAccessory,
+            deviceConfig,
+            Accessory.Categories.TEMPERATURE_SENSOR,
+            Service.TemperatureSensor
+        );
+        this.statusArr = deviceConfig.status;
+
+        this.humidityService = this.homebridgeAccessory.getService(Service.HumiditySensor);
+        if (this.humidityService) {
+            this.humidityService.setCharacteristic(Characteristic.Name, deviceConfig.name);
+        } else {
+            this.humidityService = this.homebridgeAccessory.addService(Service.HumiditySensor, deviceConfig.name);
+        }
+
+        this.refreshAccessoryServiceIfNeed(this.statusArr, false);
+    };
+
+    /**
+     * init Or refresh AccessoryService
+     * @param {*} stateArr 
+     * @param {*} isRefresh 
+     */
+    refreshAccessoryServiceIfNeed(stateArr, isRefresh) {
+        this.isRefresh = isRefresh;
+        for (const statusMap of stateArr) {
+            if (statusMap.code === 'va_temperature') {
+                const temperature = statusMap.value / 10;
+                this.normalAsync(this.service, Characteristic.CurrentTemperature, temperature);
+            } else if (statusMap.code === 'humidity_value') {
```

Comment:
Tuya Return's Status code as va_humidity https://github.com/macecchi/tuya-homebridge/pull/2

Reply:
Thanks @dipakparmar, my sensor weirdly reports as `humidity_value`, but I've added your contribution to make it work for both codes :)